devcheck

```diff
@@ -1,4 +1,5 @@
-#!/bin/sh
+#!/bin/bash
+
```

Comment:
Is the convention to have a blank line between hash-bang and license? I've generally been running them together.

(Thanks for catch on the actual binary, BTW!)

Reply:
I don't know if there's a convention. Easier to read IMO. e.g. the generated samples add a space 😋